docs/dev/news/2608.fix

```diff
@@ -0,0 +1 @@
+Labels should not be compiled in the case of variable declarations
```

Comment:
```suggestion
Labels are no longer compiled in the case of variable declarations
```

Phrase as a description of the current behavior after the change rather than a timeless assertion about how things "should" be :)